includes/cf/radarr-anime.md

```diff
@@ -1,38 +1,38 @@
+    | [{{ radarr['cf']['anime-web-tier-06-fansubs']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#anime-web-tier-06-fansubs)                   |       {{ radarr['cf']['anime-web-tier-06-fansubs']['trash_scores']['default'] }}        | {{ radarr['cf']['anime-web-tier-06-fansubs']['trash_id'] }}          |
+    | [{{ radarr['cf']['remux-tier-01']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#remux-tier-01)                                           | :warning: {{ radarr['cf']['remux-tier-01']['trash_scores']['anime-radarr'] }} :warning: | {{ radarr['cf']['remux-tier-01']['trash_id'] }}                      |
+    | [{{ radarr['cf']['remux-tier-02']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#remux-tier-02)                                           | :warning: {{ radarr['cf']['remux-tier-02']['trash_scores']['anime-radarr'] }} :warning: | {{ radarr['cf']['remux-tier-02']['trash_id'] }}                      |
+    | [{{ radarr['cf']['remux-tier-03']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#remux-tier-03)                                           | :warning: {{ radarr['cf']['remux-tier-03']['trash_scores']['anime-radarr'] }}:warning:  | {{ radarr['cf']['remux-tier-03']['trash_id'] }}                      |
+    | [{{ radarr['cf']['web-tier-01']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#web-tier-01)                                               |  :warning: {{ radarr['cf']['web-tier-01']['trash_scores']['anime-radarr'] }}:warning:   | {{ radarr['cf']['web-tier-01']['trash_id'] }}                        |
+    | [{{ radarr['cf']['web-tier-02']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#web-tier-02)                                               |  :warning: {{ radarr['cf']['web-tier-02']['trash_scores']['anime-radarr'] }}:warning:   | {{ radarr['cf']['web-tier-02']['trash_id'] }}                        |
+    | [{{ radarr['cf']['web-tier-03']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#web-tier-03)                                               |  :warning: {{ radarr['cf']['web-tier-03']['trash_scores']['anime-radarr'] }}:warning:   | {{ radarr['cf']['web-tier-03']['trash_id'] }}                        |
+    | [{{ radarr['cf']['anime-raws']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#anime-raws)                                                 |               {{ radarr['cf']['anime-raws']['trash_scores']['default'] }}               | {{ radarr['cf']['anime-raws']['trash_id'] }}                         |
+    | [{{ radarr['cf']['anime-lq-groups']['name'] }}](/Radarr/Radarr-collection-of-custom-formats/#anime-lq-groups)                                       |            {{ radarr['cf']['anime-lq-groups']['trash_scores']['default'] }}             | {{ radarr['cf']['anime-lq-groups']['trash_id'] }}                    |
```

Comment:
even if they use the default score; should we not still have the redundant anime-radarr scoring in the CF in case the default is changed there's no negative impact?

Reply:
Change of plan - we'll look again at this another time.